socket.template.ini

```diff
@@ -0,0 +1,171 @@
+
+[build]
+env = USER, TMPDIR, PWD
+flags = -O3
+headless = false
+name = "PlayerchainDemo"
+output = "build"
+copy = "dist"
+; script = "node ...."
+
+
+[build.script]
+forward_arguments = false
+
+
+[build.watch]
+; sources[] = "src"
+
+
+[webview]
+root = "/"
+watch = false
+; headers[] = "Cross-Origin-Opener-Policy: same-origin"
+; headers[] = "Cross-Origin-Embedder-Policy: require-corp"
+
+
+
+[webview.watch]
+reload = false
+; service_worker_reload_timeout = 500
+
+
+[webview.navigator.mounts]
+; $HOST_HOME/directory-in-home-folder/ = /mount/path/in/navigator
+; $HOST_CONTAINER/directory-app-container/ = /mount/path/in/navigator
+; $HOST_PROCESS_WORKING_DIRECTORY/directory-in-app-process-working-directory/ = /mount/path/in/navigator
+
+
+[webview.navigator.policies]
+; allowed[] = "https://*.example.com/*"
+
+
+[permissions]
+allow_fullscreen = true
+; allow_microphone = true
+; allow_camera = true
+; allow_user_media = true
+; allow_geolocation = true
+; allow_notifications = true
+; allow_sensors = true
+allow_clipboard = true
+; allow_bluetooth = true
+; allow_data_access = true
+; allow_airplay = true
+; allow_hotkeys = true
+
+
+[debug]
+flags = "-g"
+
+
+[meta]
+bundle_identifier = "com.playmint.playerchaindemo"
+application_protocol = "playerchaindemo"
+file_limit = 1024
+lang = "en-us"
+maintainer = "Playmint"
+title = "PlayerchainDemo"
+<% if (process.env['VERSION']) { %>
+; NOTE: Socket doesn't properly support semver versioning, it will fail to build if there are any alpha numeric chars
+version = "<%= process.env['VERSION'].split('-')[0] %>"
+title = "PlayerchainDemo v:<%= process.env['VERSION'] %>"
+<% } else { %>
+version = "0.0.1"
+title = "PlayerchainDemo v:0.0.1-dev"
```

Comment:
just curious: why do we give the dev version a different `title`?

Reply:
Annoyingly socket.ini doesn't allow alpha characters in the version number even though it says it's 'semver'! So I'm putting the whole version number including alpha characters in the title as a means to get to the whole version string. 

The fallback version number could have a title of v:0.0.1 without the -dev suffix, I was just simulating how the title maybe wouldn't match what's in the version field.